aws/table_aws_inspector_assessment_template.go

```diff
@@ -220,6 +226,40 @@ func getAwsInspectorAssessmentTemplateTags(ctx context.Context, d *plugin.QueryD
 	return op, nil
 }
 
+// API call for fetching event subscriptions
+func getAwsInspectorAssessmentEventSubscriptions(ctx context.Context, d *plugin.QueryData, h *plugin.HydrateData) (interface{}, error) {
+	logger := plugin.Logger(ctx)
+	logger.Trace("getAwsInspectorAssessmentTemplateEventSubscriptions")
+
+	var region string
+	matrixRegion := plugin.GetMatrixItem(ctx)[matrixKeyRegion]
+	if matrixRegion != nil {
+		region = matrixRegion.(string)
+	}
+
+	assessmentTemplateArn := *h.Item.(*inspector.AssessmentTemplate).Arn
+
+	// Create Session
+	svc, err := InspectorService(ctx, d, region)
+	if err != nil {
+		return nil, err
+	}
+
+	// Build the params
+	params := &inspector.ListEventSubscriptionsInput{
+		ResourceArn: &assessmentTemplateArn,
+	}
+
+	// Get call
+	op, err := svc.ListEventSubscriptions(params)
```

Comment:
@davidhammturner Can you please try using the `ListEventSubscriptionsPages` method to ensure that we get all of the subscriptions? There's an example in https://github.com/turbot/steampipe-plugin-aws/blob/main/aws/table_aws_iam_role.go#L254.